imports/ui/components/common/PanelListItem.jsx

```diff
@@ -2,37 +2,39 @@ import React from 'react'
 import IconButton from './IconButton.jsx'
 
 export default class PanelListItem extends React.Component{
-  constructor(){
-    super();
-    this.state = {
-      isEditing: false
-    }
-
-  }
 
   text() {
-    var isEditing = this.state.isEditing;
+    var isEditing = this.props.isEditing;
     var isEditable = this.props.isEditable;
 
     if(isEditable)
       return (isEditing) ? this.props.editable : this.props.text;
     else
       return this.props.text;
   }
+
   render(){
-    var isEditing = this.state.isEditing;
+    var isEditing = this.props.isEditing;
     var isEditable = this.props.isEditable;
+    var displayColor = "#A0A0A0";
+    var onMouseOverColor="#ff9100";
+
+    let icon = <IconButton
+                  style={{float: "right"}}
+                  icon={this.props.icon}
+                  displayColor= {displayColor}
+                  onMouseOverColor= {onMouseOverColor}
+                  onButtonClick={this.props.iconClick}/>;
+
     return(
-        <li className={this.props.type}>
-          <a href="#">
+        <li className={this.props.displayType}>
+          <a href="#"
+             style={ isEditable? {cursor: 'pointer'} : {cursor: 'default'}}
```

Comment:
Nice UI design in reactively styling the cursors to mark whether isEditable or not